specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/examples/FirewallPolicyQuerySignatureOverrides.json

```diff
@@ -0,0 +1,68 @@
+{
+  "parameters": {
+    "api-version": "2021-02-01",
```

Comment:
Could you update the api version in examples?

Reply:
Done.